plan/physical_plan_builder.go

```diff
@@ -141,6 +141,9 @@ func (p *DataSource) convert2TableScan(prop *requiredProperty) (*physicalPlanInf
 		DBName:              p.DBName,
 		physicalTableSource: physicalTableSource{client: client},
 	}
+	ts.tp = "TableScan"
```

Comment:
I will remove the constants in future. The type constant won't be used frequently. It's better to use string literal.